pandas/_libs/lib.pyx

```diff
@@ -67,6 +67,7 @@ import tslib
 from tslib import NaT, Timestamp, Timedelta
 import interval
 from interval import Interval
+from properties import AxisProperty, cache_readonly  # noqa
```

Comment:
you may be able to remove these imports entity and just change where they r called from

Reply:
Sure.  It looks like the only direct import of `lib.cache_readonly` is by `util._decorators`, and other imports get it from there.  So that would only require changing it in one place.  And `lib.AxisProperty` is only used in `core.generic`, so also an easy change.  Pls confirm before I make this new change.

Reply:
yep jose would be good